tfhe/src/lib.rs

```diff
@@ -50,8 +56,6 @@
 #![allow(clippy::explicit_iter_loop)]
 // End allowed pedantic lints
 
-// Nursery lints
-#![warn(clippy::nursery)]
 // The following lints have been temporarily allowed
```

Comment:
Why move?

I believe the following allowed lints are all nursery (or used to be)

Reply:
I move because there were breaking changes in terms of which lint belong to which, so enable all warnings first, then re allow to make sure we don't have breaking stuff later